helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -702,6 +702,13 @@ fluentd:
     # https://docs.fluentd.org/input/tail#pos_file-highly-recommended
     posFilePrefix: /var/log/splunk-fluentd
 
+    # Specify the interval of refreshing the list of watch file. Defaults to 60 (seconds)
+    refreshInterval: 60
```

Comment:
Comment out the new optionals since they have the same default values set in [helm-charts/splunk-otel-collector/templates/configmap-fluentd.yaml](https://github.com/signalfx/splunk-otel-collector-chart/pull/1010/files#diff-b67ef7e00d36603215656a67dac7a3fac856bdfd7b25c8c7000e2ac954f175ae). This way we don't set the default in two places.